StaticAnalyzer/views/android/static_analyzer.py

```diff
@@ -548,8 +549,9 @@ def get_app_name(app_path, app_dir, tools_dir, is_apk):
     
     data = ""
     if is_apk:
-        output_dir = os.path.join(app_dir, "apktool_out")
-        strings_file = os.path.join(output_dir, "res/values/strings.xml")
+        a = apk.APK(app_path)
```

Comment:
Is this always guaranteed to return a real name.?

Reply:
yep all the test I have done also with apk which did not use app_name were Ok

Reply:
for me yes